crates/store/src/db/sql/utils.rs

```diff
@@ -86,20 +86,32 @@ pub fn column_value_as_u64<I: rusqlite::RowIndex>(
     Ok(value as u64)
 }
 
+pub fn read_from_blob_column<I, T>(row: &rusqlite::Row<'_>, index: I) -> rusqlite::Result<T>
```

Comment:
I know I added this, but we should add a brief doc comment mentioning what this does.

Reply:
We did something [similar in pathfinder](https://github.com/eqlabs/pathfinder/blob/79a26781e85b03fac548c7da387a5863df95104a/crates/storage/src/params.rs#L154-L395), but we did this as a trait extension for all our types.

I'll probably try something similar here at some point, it makes things quite ergonomic and type safe because you can just go

```rust
row.get_account_id(0)
```